test/unit/update_versions_file_test.rb

```diff
@@ -44,7 +44,7 @@ class UpdateVersionsFileTest < ActiveSupport::TestCase
       end
 
       should "include platform release" do
-        expected_output = "rubyrubyruby 0.0.1-jruby,0.0.1 qw212r\n"
+        expected_output = "rubyrubyruby 0.0.1,0.0.1-jruby qw212r\n"
```

Comment:
info response should not change because of this. `gem_lines` is only being used to generate versions file. @deivid-rodriguez please verify this anyway.

I use the following steps to verify checksum:
```
$ rake compact_index:update_versions_file
$ cat lib/versions_verify_checksum.rb
file = File.open("/home/aditya/rubygems.org/log/versions-updated.log", 'w')
File.readlines("/home/aditya/rubygems.org/config/versions.list").drop(2).each do |line|
  name, versions, info_checksum = line.split(" ")
  calc_info_checksum = Digest::MD5.hexdigest(CompactIndex.info(GemInfo.new(name).compact_index_info))
  if calc_info_checksum != info_checksum
    puts "#{name} #{info_checksum} #{calc_info_checksum}"
    file.write("#{name} #{info_checksum} #{calc_info_checksum}\n")
  end
end
file.close
$ rails runner lib/versions_verify_checksum.rb
```
Haven't done this so far for compact_index update.

Reply:
Ok, I'll double check it!

Reply:
I'm not sure if I need to test anything else here, but the gems I used for testing have a lot of platform specific versions, and I didn't see any issues with missing checksums so I believe it's fine.

If I understand correctly, this is a change in the versions file formatting, but it doesn't change the `/info` endpoint, so it should affect checksums. Is that correct?